xdcs-frontend/src/app/view/task-creation/task-creation.component.html

```diff
@@ -74,15 +74,65 @@ <h2>Resources</h2>
 </ng-template>
 
 <ng-template #parameters>
-  <ng-container *ngIf="['opencl', 'cuda'].includes(deployment.config.type); else noParameters"
+  <ng-container *ngIf="['opencl', 'cuda'].includes(deployment.config.type)"
```

Comment:
jak typem jest `opencl` lub `cuda` to wyświetlą się zarówno parametry jak i wiadomość, że ich nie ma